docker/test/integration/features/https.feature

```diff
@@ -1,3 +1,5 @@
+# Failing feature, needs to be fixed
```

Comment:
a `TODO` or `FIXME` in the comment would make it easier to find later

Reply:
Update in cdb014f306f614bfad5e6120f089ab821c904677